packages/contracts-bedrock/scripts/L2Genesis.s.sol

```diff
@@ -571,6 +576,26 @@ contract L2Genesis is Script {
         IGasPriceOracle(Predeploys.GAS_PRICE_ORACLE).setIsthmus();
     }
 
+    /// @notice This predeploy is following the safety invariant #1.
+    function setFeeSplitter(Input memory _input) internal {
+        address impl = _setImplementationCode(Predeploys.FEE_SPLITTER);
+        vm.startPrank(_input.opChainProxyAdminOwner);
+        // Initialize the implementation with address(0) for addresses
+        IFeeSplitter(payable(impl)).initialize(
```

Comment:
It would be less confusing if these were initialized to zero values. Sometimes people confuse the implementation and the proxy